index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;600;700;800;900&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Poppins:wght@200;300;500;600;700;800;900&display=swap" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+
+  <section class="header">
+    <div class="header-upper">
+      <ul>
+        <li><a href="#"><i class="fa fa-facebook" ></i></a></li>
+        <li><a href="#"><i class="fa fa-twitter" ></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav>
+
+
+      <div class="logo">
+        <a href="index.html"><img src="./images/new-logo.png" alt="Logo"></a>
+      </div>
+      <div class="menu-items">
+        <ul class="nav-link">
+          <li class="link"><a href="about.html" id="aboutit">About</a></li>
+          <li class="link"><a href="#">Program</a></li>
+          <li class="link"><a href="#">Join</a></li>
+          <li class="link"><a href="#">Sponsers</a></li>
+          <li class="link"><a href="#">News</a></li>
+
+        </ul>
+        <div class="header-button"><button type="button" class="button-header">G Gathering</button></div>
+
+      </div>
+    </nav>
+  </section>
```

Comment:
Kindly try to match the `Behance` design for the header section in the mobile version on the index page. the `font size` needs to be smaller and Please change the `section` tag with the `header` tag for semantic purposes.
| Your design | Behance design |
| --- | --- |
| ![screencapture-raw-githack-monika-soam-Marketing-Conference-About-Page-index-html-2022-03-09-23_44_21](https://user-images.githubusercontent.com/24703171/157543638-fdd8ea8b-d953-4c18-85ba-0653cb632a09.png) | ![image](https://user-images.githubusercontent.com/24703171/157543397-36c073d7-2ab0-423d-b20a-6e328a9919d8.png) |

Reply:
This is fixed